Oxford University Press.js

```diff
@@ -113,26 +154,27 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "24 July 2003",
-				"ISBN": "978-0-19-322103-1",
-				"abstractNote": "Tutor Easy Piano accompaniments to selected pieces in Flute Time 1",
+				"date": "2003-07-24",
+				"ISBN": "9780193221031",
+				"abstractNote": "Piano accompaniments to selected pieces in Flute Time 1",
 				"libraryCatalog": "Oxford University Press",
 				"numPages": "32",
-				"publisher": "Flute Time",
-				"attachments": [],
-				"tags": [
-					"Flute",
-					"Music",
-					"Woodwind & Brass"
+				"publisher": "Oxford University Press",
```

Comment:
Well, normally I think that the place of publication is a strange (antique) metadata field and I am not sure what the purpose is nowadays. However, it is a widely used field in citation styles, thus it might be a user-friendly idea to give some info when possible.
1. If the place of publication should be derived somehow from the book, then this value might change slightly and there are also some variations in bibliographic entries in library catalogues. Thus, it wouldn't be always correct to save a fixed value here.
2. We can see the place of publication as an additional info to the publisher, which might useful to distinguish or specify several publisher's with the same name or give some context. Then, I guess it is save to add one general value like "Oxford".
3. The production (typesetting etc.) is sometimes stated to be in India.
4. I doubt that for distribution, sales the place of publication is important to know nowadays...
5. The place of publication (especially the country) might matter in some legal affairs.
6. The country code in the ISBN is AFAIS always `0` which stands for the UK and US and other English speaking regions.

Possible values I have seen:
   * Oxford
   * Oxford, New York
   * Oxford and others

But this could be much more detailed (e.g. published in US, printed in UK, and typesetted in India):

![oxford-imprint](https://cloud.githubusercontent.com/assets/5199995/21718411/97e18852-d417-11e6-9b4a-b40315fa7be5.png)
![oxford-imprint-2](https://cloud.githubusercontent.com/assets/5199995/21718651/1fb6b4b8-d419-11e6-82d7-6063c9940d47.png)